components/NavBar.jsx

```diff
@@ -0,0 +1,81 @@
+import {
+  Avatar,
+  Box,
+  Flex,
+  HStack,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuDivider,
+  MenuItem,
+  MenuList,
+  Text,
+  useColorModeValue,
+  VStack,
+} from '@chakra-ui/react';
+import { FiChevronDown, FiMenu, FiBell } from 'react-icons/fi';
+
+function NavBar({ onOpen, ...rest }) {
+  return (
+    <Flex
+      ml={{ base: 0, md: 60 }}
+      px={{ base: 4, md: 4 }}
+      height="20"
+      alignItems="center"
+      bg={useColorModeValue('white', 'gray.900')}
```

Comment:
is that for dark mode ?

Reply:
Ah yes, soon for DarkMode features. I will fix it later. while I make an issue for it first #3